plugin/src/main/kotlin/spp/jetbrains/sourcemarker/status/LiveStatusManager.kt

```diff
@@ -74,6 +77,17 @@ object LiveStatusManager : SourceMarkEventListener {
 
     private val log = LoggerFactory.getLogger(LiveStatusManager::class.java)
     private val activeStatusBars = CopyOnWriteArrayList<LiveInstrument>()
+    private val logData = ConcurrentHashMap<String?, List<*>>()
+
+    fun getLogData(inlayMark: InlayMark?): List<*> {
```

Comment:
Why `InlayMark?`?  Seems unnecessary

Reply:
Removed ?

Reply:
> Still getting a waiting message before the first log shows: ![still-waiting](https://user-images.githubusercontent.com/3278877/153286820-9c7b79b9-202a-407e-a80d-fc6918aa5dfe.gif)
> 
> Instead of waiting, it should show the last received message.

fixed this. but had to do json conversion as the logs added to the ListModel are in json and the occuredAt field is a Kotlin Instant converting to java.time.Instant looks bit weird. Please check the code

((kotlinx.datetime.Instant) Json.decodeValue(event.getData(), LiveLogHit.class).getOccurredAt()).getValue$kotlinx_datetime();